### ISSUE TITLE
about input shape in own dataset

### ISSUE THREAD

**Issue by WJShengD** [JiahuiYu/generative_inpainting#215]

**Comment by WJShengD** [comment]
Can this network be directly used in 64*64*3 input size? and I input the 64*64*3 image directly into the network to train the network, but the loss value is Nan in the same place. Finally, what is the learning rate in inapint.yml? Your reply will be my pleasure. Thank you

**Comment by Yukariin** [comment]
@WJShengD just read the [config](https://github.com/JiahuiYu/generative_inpainting/blob/master/inpaint.yml#L55) duh

**Comment by JiahuiYu** [comment]
@Yukariin Thanks for your help and contribution!
